client/src/main/java/org/apache/rocketmq/client/impl/factory/MQClientInstance.java

```diff
@@ -839,8 +839,8 @@ public boolean updateTopicRouteInfoFromNameServer(final String topic, boolean is
                         log.warn("updateTopicRouteInfoFromNameServer Exception", e);
                     }
                 } catch (RemotingException e) {
-                    log.error("updateTopicRouteInfoFromNameServer Exception", e);
-                    throw new IllegalStateException(e);
+                    log.error("updateTopicRouteInfoFromNameServer Exception, namesrvAddr: " + clientConfig.getNamesrvAddr(), e);
```

Comment:
Please log with formatter, not string concatenation. See https://medium.com/@mohanakrishnakavali/logging-efficiency-in-java-string-concatenation-vs-string-format-efa02055ddae